### ISSUE TITLE
Proper LPC43XX support to port to rad1o

### ISSUE THREAD

**Issue by brabo** [insane-adding-machines/frosted#43]
Port frosted to rad1o, examine/enhance libopencm3 support for LPC43XX.
For one, the hackrf libopencm3 branch has defines that are lacking in libopencm3 such as
CGU_PLL1_CTRL_PD_MASK
and others.

**Comment by danielinux** [comment]
can we merge in hackrf changes into our branch?

**Comment by brabo** [comment]
@danielinux yes, that is the idea. Atm frosted runs with idling LEDs 1-3. LED4 does not work yet.
Next up would be basic uart ^^
